src/root/components/Container/index.tsx

```diff
@@ -66,13 +68,18 @@ function Container(props: Props) {
       >
         {(heading || actions) && (
           <Header
-            className={_cs(styles.header, headerClassName)}
+            className={_cs(styles.headerClassName)}
```

Comment:
Double check if styles.header was not being used?
```suggestion
            className={styles.headerClassName}
```